tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "sourceMap": true,
     "strict": true,
     "skipLibCheck": true,
-    "moduleResolution": "node",
+    "moduleResolution": "bundler",
```

Comment:
Changed `moduleResolution` to `bundler` to support Storybook 10's package `export` patterns. This shouldn't impact build.